### DISCUSSION TITLE
Classic Scatterplot plugin

### DISCUSSION THREAD

**Discussion by fccoelho** [apache/superset#19087] [Ideas]
**Is your feature request related to a problem? Please describe.**
I am looking for a simple  X Y scatterplot in superset but it seems that none are available as of release 1.4.0, the one I use.

**Describe the solution you'd like**
Browsing your repository, I found this:
https://apache-superset.github.io/superset-ui/?path=/story/chart-plugins-preset-chart-xy-scatterplot--basic
That although basic (does not have option to add trendlines, elipses, heatmaps, etc.), Would be usable to someone that really needs something like this.

**Describe alternatives you've considered**
There are a number of quite nice of scatterplot implementations here:
https://echarts.apache.org/examples/en/#chart-type-scatter

**Comment by srinify** [comment]
Moved to Discussions since this is a feature request and not a bug!

**Comment by raags** [comment]
There's a legacy chart type called, Bubble Chart - I think that's what you're looking for.

**Reply by fccoelho** [reply]
Not really, because it is too rigid, it requires 4 variables, Entity, X, Y, and Bubble size ( which can be a metric) so it does't work for simple Scatterplots that only wants to plot X against Y.

**Reply by raags** [reply]
Entity is optional, but there is also "Time-series Scatter Plot"

**Reply by fccoelho** [reply]
Just tried. Entity cannot be empty. My simple dataset only has X and Y  and a date index
But I managed to make it work. Not Ideal though, and result is not great. One annoying thing is that it forces you to use a metric rather than plain column values for both X and Y and you can't repeat the metric for bubble size(which is not optional). 

It is not as intuitive as the interfaces of Scatter plots implemented in widely used libraries such as matplotlib, Seaborn, or Pandas.

**Reply by fccoelho** [reply]
Time-series Scatter plot requires that time is the x axis, so you can't have one variable against the other.

**Reply by raags** [reply]
Oh yes, I confused entity with Series - I agree, a echarts implementation with less restrictions would be good.
